packages/core/src/editor/query.tsx

```diff
@@ -260,14 +288,35 @@ export function QueryMethods(state: EditorState) {
             return false;
           }
         },
+        serialize: () => this.serialise(node),
       };
     },
 
+    /**
+     * Given a Node, it serializes it to its node data. Useful if you need to compare state of different nodes.
+     *
+     * @param node
+     */
+    getSerializedNode(node: Node): SerializedNode {
+      return serializeNode(node.data, options.resolver);
+    },
+
+    /**
+     * Returns all the `nodes` in a serialized format
+     */
+    getSerializedNodes(): SerializedNodes {
```

Comment:
`getSerializedNodesFromState` to differenciate from the one above? it's quite of a mouthful though

Reply:
Or perhaps let's rename `getSerializedNode` with the "parse" prefix similar to the other parse queries since we are not actually getting a state value but rather transforming an object into a different form. So maybe like `parseSerializedNodeFromNode`. 

It's mouthful but I like that it's descriptive. Maybe we can create a collection similar to NodeHelpers, eg: `query.parse.node.fromSerialised()` or something like that 😆

Reply:
yeah I like the idea of the static node helpers, I think it will clean up some clutter. Buc yeah, let's keep it to another PR.
Ok let's change it to `parseSerializedNodeFromNode`, I think it makes sense.